test/parallel/test-crypto-hmac.js

```diff
@@ -8,6 +8,14 @@ if (!common.hasCrypto) {
 }
 const crypto = require('crypto');
 
+// Test for binding layer robustness
+{
+  const binding = process.binding('crypto');
+  const h = new binding.Hmac();
+  // Fail to init the Hmac with an algorithm.
```

Comment:
Nit: It took me a little bit to realize this comment was saying "We fail to init the Hmac with an algorithm, so an attempt to update should throw." I read it as "We're going to fail trying to init the Hmac with an algorithm" and I was all like, "What? I do not understand this test."

Maybe expand the comment a bit to something like: "Hmac.update() should throw if Hmac has not been initialized with an algorithm" or something like that?